droidlet/tools/hitl/utils/hitl_recorder.py

```diff
@@ -79,38 +76,6 @@ class JobStat(Enum):
     MODEL_LOSS = "model_loss"
 
 
-# statastics that all jobs have
```

Comment:
I deleted these part cause the current version will not create stat ahead of time, so no need to have `STAT_FOR_ALL` and `STAT_JOB_PAIR` sets.